movement/watch_faces/complication/countdown_face.c

```diff
@@ -203,15 +245,6 @@ bool countdown_face_loop(movement_event_t event, movement_settings_t *settings,
         case EVENT_BACKGROUND_TASK:
             ring(state);
             break;
-        case EVENT_ALARM_LONG_PRESS:
```

Comment:
I know this doesn't mirror the original firmware, but I think we should keep this quick reset around. Otherwise it takes a very large number of button presses to input a timer that is less than what is currently showing. Or we implement hold to quickly advance the current setting, which is more in line with most timer behavior on other watches. If you wanted to try to tackle that, [the set time face](https://github.com/joeycastillo/Sensor-Watch/blob/main/movement/watch_faces/settings/set_time_face.c#L105) shows how that can be done. If that seems like too much to chew off, I'd just restore this block for now.